packages/template-typescript-minimal/app/pages/home.tsx

```diff
@@ -4,171 +4,96 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import React, {useEffect, useState} from 'react'
-import {useQuery} from '@tanstack/react-query'
-
+import React from 'react'
 import {useApplicationExtensions} from '@salesforce/pwa-kit-extension-sdk/react'
 
-import HelloTS from '../components/hello-typescript'
-import HelloJS from '../components/hello-javascript'
-
-interface Props {
-    value: number
-}
-
 const style = `
-body {
-    background: linear-gradient(-45deg, #e73c7e, #23a6d5, #ee7752);
-    background-size: 400% 400%;
-    height: 100vh;
-}
-@keyframes gradient {
-    0% {
-        background-position: 0% 50%;
-    }
-    50% {
-        background-position: 100% 50%;
-    }
-    100% {
-        background-position: 0% 50%;
-    }
-}
-@keyframes fade {
-  0% { opacity: 0 }
-  100% { opacity: 1 }
-}
-.fade-in {
+.content {
```

Comment:
Fixed and constrained the css selectors to avoid the styles leaking to other unexpected elements.